frontend/src/components/Admin/TeamEvent/TeamEventCreditReview.tsx

```diff
@@ -24,7 +26,8 @@ const TeamEventCreditReview = (props: {
   const approveCreditRequest = (teamEventAttendance: TeamEventAttendance) => {
     const updatedTeamEventAttendance = {
       ...teamEventAttendance,
-      pending: false
```

Comment:
Why don't we keep `pending` for now still? My reasoning is, if some of these changes break something on IDOL, we can always rollback the changes, and our DB will still be compatible with the old backend.